src/table/Root.jsx

```diff
@@ -24,7 +24,3 @@ export function render(reactRoot, props) {
 export function teardown(reactRoot) {
   reactRoot.unmount();
 }
-
-export function mount() {
```

Comment:
Hi @vcellu, I see that you added the mount function. Is it okay to remove it? I do not see that it is needed,

Reply:
it is defined in the root.native file, it needs to run in the native case. In the web case it just does nothing and thus no condition needs to check the environment in the useReactRoot hook. You should probably revert this